public/app/percona/shared/services/advisors/Advisors.utils.ts

```diff
@@ -0,0 +1,20 @@
+import { Advisor, CategorizedAdvisor } from './Advisors.types';
+
+export const groupAdvisorsIntoCategories = (advisors: Advisor[]): CategorizedAdvisor => {
+  const result: CategorizedAdvisor = {};
+
+  if (advisors.length > 0) {
```

Comment:
The length might not be necessary here. If there aren't any elements the forEach wouldn't run.

Reply:
True, good point